content/en/docs/concepts/containers/pod-overhead.md

```diff
@@ -0,0 +1,46 @@
+---
+reviewers:
+- tallclair
+- dchen1107
+- egernst
+title: Pod Overhead
+content_template: templates/concept
+weight: 20
+---
+
+{{% capture overview %}}
+
+{{< feature-state for_k8s_version="v1.16" state="alpha" >}}
+
+This page describes pod Overhead, as defined in RuntimeClass and pod.Spec, and mechanisms for applying this feature.
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+## Pod Overhead
+
+Pod Overhead is a feature for applying an overhead to a pods definition at admission time depending on the RuntimeClass being utilized.
```

Comment:
This line feels like a great fit for the overview section.

Also, how about adding a hyperlink to “admission”:
```suggestion
Pod Overhead is a feature for applying an overhead to a pods definition at [admission](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#what-are-admission-webhooks) time, depending on the RuntimeClass being utilized.
```

If the link doesn't feel right, maybe I can find or tweak another page that explains _admission_ as a Kubernetes concept.